test/components/legendTests.ts

```diff
@@ -350,3 +350,183 @@ describe("Legend", () => {
     svg.remove();
   });
 });
+
+describe("InterpolatedColorLegend", () => {
```

Comment:
the name is the file is now super confusing because of the shared namespaces...  We probably want to place this in a separate file.